lib/model/IngredientsAnalysisTags.dart

```diff
@@ -57,42 +87,51 @@ class IngredientsAnalysisTags {
     }
 
     switch (ingredientsAnalysisTags.veganStatus) {
-      case VeganStatus.IS_VEGAN:
+      case VeganStatus.VEGAN:
         result.add('en:vegan');
         break;
-      case VeganStatus.IS_NOT_VEGAN:
+      case VeganStatus.NON_VEGAN:
         result.add('en:non-vegan');
         break;
-      case VeganStatus.MAYBE:
+      case VeganStatus.MAYBE_VEGAN:
         result.add('en:maybe-vegan');
         break;
+      case VeganStatus.VEGAN_STATUS_UNKNOWN:
+        result.add('en:vegan-status-unknown');
+        break;
```

Comment:
Perhaps we should say `UNKNOWN` if we don't know. But I'm not sure if it's strictly the same in OFF "not to know" and "know that we don't know".